arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,937 @@
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was created."
+        },
+        "serviceBusEndpoint": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+          },
+          "type": "string",
+          "description": "Endpoint you can use to perform ServiceBus operations."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The Id of the Azure subscription associated with the namespace."
+        },
+        "critical": {
+          "type": "boolean",
+          "description": "Whether or not the namespace is set as Critical."
+        },
+        "namespaceType": {
+          "type": "string",
+          "description": "The namespace type.",
+          "enum": [
+            "Messaging",
+            "NotificationHub",
+            "EventHub"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Namespace properties."
+    },
+    "RegenerateKeysParameters": {
+      "properties": {
+        "Policykey": {
+          "type": "string",
+          "description": "Key that needs to be regenerated .",
+          "enum": [
+            "PrimaryKey",
+            "SecondayKey"
+          ],
+          "x-ms-enum": {
+            "name": "Policykey",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Parameters supplied to the Regenerate Namespace Auth Rule."
+    },
+    "NamespaceCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Namespace data center location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Namespace tags."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NamespaceProperties",
+          "description": "Properties of the Namespace."
+        }
+      },
+      "required": [
+        "location",
+        "properties"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Namespace operation."
+    },
+    "NamespaceResource": {
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Kind of the Namespace.",
+          "enum": [
+            "Messaging",
+            "EventHub"
+          ],
+          "x-ms-enum": {
+            "name": "kind",
+            "modelAsString": true
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/ARMSku",
+          "description": "Sku of the Namespace."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
```

Comment:
Sibling properties of the `$ref` should actually be in the referenced definition. (Extra JSON Reference properties will be ignored)